sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsBufferCatalog.scala

```diff
@@ -298,4 +299,6 @@ object RapidsBufferCatalog extends Logging with Arm {
 
   /** Remove a buffer ID from the catalog and release the resources of the registered buffer. */
   def removeBuffer(id: RapidsBufferId): Unit = singleton.removeBuffer(id)
+
+  def getDiskBlockManager(): RapidsDiskBlockManager = diskBlockManager
```

Comment:
I had commented on this nit in the previous PR, but it is minor and not a blocker.
```suggestion
  def getDiskBlockManager: RapidsDiskBlockManager = diskBlockManager
```